src/api/ledger/transaction.js

```diff
@@ -78,11 +78,22 @@ function getTransactionAsync(identifier: string, options: TransactionOptions,
     }
   }
 
+
+  /* eslint-disable no-unused-vars, handle-callback-err */
```

Comment:
It's bad practice because getLedgerSequence could change in the future in unexpected ways. We should always follow best practices and not take shortcuts. Generally I think there are very few cases where disabling eslint rules is justified.